src/compton2/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+#-----------------------------*-cmake-*----------------------------------------#
+# file   src/compton/CMakeLists.txt
+# author Andrew Till
```

Comment:
I think merging this into `compton` is best.  Let me know what you think.

Reply:
I'd like to keep them separate so that we could remove the entire `compton` directory later if we wanted. Otherwise, we'd have to be very careful about what was a standalone functionality and what was a wrapper.